src/test/resources/hudson/plugins/ec2/UnixData.yml

```diff
@@ -12,6 +12,11 @@ jenkins:
             type: "T2Micro"
             remoteFS: "/home/ec2-user"
             mode: "NORMAL"
+            spotConfig:
```

Comment:
Can we ensure that this was set appropriately in https://github.com/jenkinsci/ec2-plugin/blob/master/src/test/java/hudson/plugins/ec2/ConfigurationAsCodeTest.java

Reply:
I added a test when it's there and another one when it's not.